.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"xyc.vscode-mdx-preview",
+		"silvenon.mdx",
+		"eamodio.gitlens",
```

Comment:
Would only add extensions useful to the repo, gitlens is a tool that doesn't benefit the repository.